spec/unit/builders/ProjectBuilder.spec.js

```diff
@@ -29,12 +28,12 @@ describe('ProjectBuilder', () => {
 
     let builder;
     let ProjectBuilder;
-    let spawnSpy;
+    let execaSpy;
 
     beforeEach(() => {
-        spawnSpy = jasmine.createSpy('spawn').and.returnValue(Q.defer().promise);
```

Comment:
The `Q` changes I think should done in the dropping Q PR : https://github.com/apache/cordova-android/pull/833, but this type of work kinda blends together.

Reply:
The old code used `superspawn` which returns `Q` Promises, so it was natural for the test stubs to return `Q` Promises too. Now that we stub `execa`, we should not return `Q` Promises from the stub anymore, as the interface provided by them is not a subset of the type returned from `execa`. That could lead to code passing the tests but failing in reality.